anitya/lib/backends/hackage.py

```diff
@@ -62,3 +62,12 @@ def get_versions(cls, project):
         regex = REGEX % {'name': project.name}
 
         return get_versions_by_regex(url, regex, project)
+
+    @classmethod
+    def check_feed(cls):
+        ## See http://hackage.haskell.org/api#recentPackages
+        ## It should be possible to query this, but I can't figure out how to
+        ## get it to give me non-html.
+        # url = 'http://hackage.haskell.org/packages/recent/revisions'
```

Comment:
Just adding `.rss` seems to do it: http://hackage.haskell.org/packages/recent.rss